src/data/pages/distance-to-supernovae-2.json

```diff
@@ -53,7 +49,7 @@
           "label": "<p>Enter the peak apparent magnitude (m) for supernova #4 to find its distance from Earth.</p>",
           "placeholder": "Enter peak magnitude",
           "answerPre": "<p>Selected: </p>",
-          "answerAccessor": "text"
+          "answerAccessor": "magnitude"
```

Comment:
I'm not sure what this change is accomplishing.  If anything wouldn't this change to like parsecs or megaLightyears or distance or something?  Or even just null?  In any case you're never using the answerAccessor in the DistanceCalculator and if some other component was using a DistanceCalculator answer for something we wouldn't want it to grab the magnitude off the answer data we'd want it to grab one of the distance measurements right?